powerauth-restful-security-javaee/src/main/java/io/getlime/security/powerauth/rest/api/jaxrs/service/v3/UpgradeService.java

```diff
@@ -142,7 +142,7 @@ public Response upgradeCommit(String signatureHeader,
         } catch (PowerAuthAuthenticationException ex) {
             throw ex;
         } catch (Exception ex) {
-            logger.warn("PowerAuth upgrade commit failed", ex);
+            logger.warn("PowerAuth upgrade commit failed, error: {}", ex.getMessage());
```

Comment:
Let's pass `ex` as a parameter to the new exception to keep the cause somewhere.